packages/queued-request-controller/src/QueuedRequestController.ts

```diff
@@ -94,6 +91,12 @@ type QueuedRequest = {
    * The origin of the queued request.
    */
   origin: string;
+
+  /**
+   * The networkClientId of the queuedRequest.
+   */
+  networkClientId: NetworkClientId;
```

Comment:
this is a good catch that it will be on the request already and don't need the SelectedNetworkController call in here.